model-api-gen-runtime/build.gradle.kts

```diff
@@ -59,3 +63,27 @@ kotlin {
 }
 
 description = "Runtime for the meta model generator"
+
+npmPublish {
```

Comment:
Why do we need to publish `@modelix/model-api-gen-runtime`?
I did not find its usage.